bugsnag-android-core/src/main/java/com/bugsnag/android/AppDataCollector.kt

```diff
@@ -49,12 +64,54 @@ internal class AppDataCollector(
         )
     }
 
+    @SuppressLint("SwitchIntDef")
+    @Suppress("DEPRECATION")
+    private fun getProcessImportance(): String? {
+        try {
+            val appInfo = ActivityManager.RunningAppProcessInfo()
+            if (VERSION.SDK_INT >= VERSION_CODES.JELLY_BEAN) {
+                ActivityManager.getMyMemoryState(appInfo)
+            } else {
+                val expectedPid = Process.myPid()
+                activityManager?.runningAppProcesses
+                    ?.find { it.pid == expectedPid }
+                    ?.let { appInfo.importance = it.importance }
```

Comment:
Since this doesn't assign `appInfo.pid` the following `if (appInfo.pid == 0)` will always early-exit.